browser/containers/Metrics.tsx

```diff
@@ -121,9 +121,9 @@ export default class Metrics extends React.Component<Props, State> {
       this.state.topContribProjectsLastYear
     );
     return (
-      <div>
+      <>
```

Comment:
Why leave these here?

Reply:
Mmm.... because I was testing and forgot. 😶 Removing